cli/src/commands/new.rs

```diff
@@ -164,6 +165,8 @@ pub fn handle_new_command(
             streams: None,
             chat: None,
         }],
+        // TODO
+        native_transfers: NativeTransfers::default(),
```

Comment:
default should be off i think

Reply:
Default is off, it will internally flip to `enabled: false` at the moment.